src/Sanitizer.php

```diff
@@ -65,9 +72,22 @@ class Sanitizer
      */
     protected $context;
 
-    public function __construct(VisitorInterface ...$visitors)
+    /**
+     * @param Behavior|VisitorInterface[] $items
+     *
+     * @todo use `__construct(Behavior $behavior, VisitorInterface ...$visitors)`
+     * (which would have been a breaking change with a PHP fatal error)
+     */
+    public function __construct(...$items)
     {
-        $this->visitors = $visitors;
+        $this->visitors = [];
+        foreach ($items as $item) {
+            if ($item instanceof VisitorInterface) {
+                $this->visitors[] = $item;
+            } elseif ($item instanceof Behavior && $this->behavior === null) {
+                $this->behavior = $item;
+            }
+        }
         $this->parser = $this->createParser();
```

Comment:
Maybe a deprecation error should be thrown in case `$this->behavior` is not defined?

Reply:
Done! ✅